examples/application_commands/sub_commands.py

```diff
@@ -1,11 +1,12 @@
 import nextcord
```

Comment:
`from nextcord import Interaction` could be used, for consistency across examples